include/xrpl/server/detail/SSLHTTPPeer.h

```diff
@@ -115,14 +115,14 @@ SSLHTTPPeer<Handler>::run()
 {
     if (!this->handler_.onAccept(this->session(), this->remote_address_))
     {
-        boost::asio::spawn(
+        ripple::util::spawn(
```

Comment:
```suggestion
        util::spawn(
```

`ripple::` can be removed